portals/publisher/src/main/webapp/site/public/locales/en.json

```diff
@@ -8565,12 +8643,30 @@
       "value": "Policies"
     }
   ],
+  "Base.Header.navbar.GlobalNavBar.global.policies": [
+    {
+      "type": 0,
+      "value": "Global Policies"
+    }
+  ],
   "Base.Header.navbar.GlobalNavBar.scopes": [
     {
       "type": 0,
       "value": "Scopes"
     }
   ],
+  "Cancel": [
+    {
+      "type": 0,
+      "value": "Cancel"
+    }
+  ],
+  "Cannot.Find.PolicyObj.For.PolicyId": [
+    {
+      "type": 0,
+      "value": "Cannot find policyObj for policyId:"
```

Comment:
"Cannot find policyObj for policyId:" This message is a bit too technical. Let's change it to be more user friendly with names identifiable by the end user.

Reply:
This is also dynamic. Changed it to 
```
 else {
  Alert.error(intl.formatMessage({
      id: 'Cannot.Find.PolicyObj.For.PolicyId',
      defaultMessage: 'Cannot find policy for Id: ',
  }) + policyId);
```

Will this be enough?